### ISSUE TITLE
[Bug] Cannot update patch

### ISSUE THREAD

**Issue by moonstarsky37** [jammsen/docker-palworld-dedicated-server#61]
### Have you read the Important information text above

- [X] Yes i did

### Current behavior

Since palworld has new patch https://steamdb.info/app/2394010/config/
I docker compose down the server and docker pull the latest image
but always show Error! App '2394010' state is 0x606 after update job.

i also git pull the latest repo

### Desired behavior

want to update successfully

### Links to screenshots

![圖片](https://github.com/jammsen/docker-palworld-dedicated-server/assets/9031339/9f639efa-a050-4636-a0df-1891890f4291)

### To Reproduce

```
docker compose down
docker pull jammsen/palworld-dedicated-server:latest
git pull
docker compose up
```

### Software setup

- OS: Ubuntu 20.04.4 LTS
- Docker:  24.0.7

### Hardware setup

- vCPU: 32
- RAM: 64G
- Disk: 1TB

**Comment by cephalization** [comment]
Same error on a fresh install after renaming my game directory

**Comment by cephalization** [comment]
I feel dumb. My fedora installation only used 16gb of my available 500gb ssd. 

It works perfectly now after growing my partition

**Comment by jammsen** [comment]
> Sorry, same result (game won't update or install), but different error
> `palworld-dedicated-server | Error! App '2394010' state is 0x202 after update job.`

Well yeah 0x202 is known for missing hdd-space 🤣 

> I feel dumb. My fedora installation only used 16gb of my available 500gb ssd.
> 
> It works perfectly now after growing my partition

Happy to hear that. Then im going to close this issue as resolved, feel free reopen again if needed.

If you like this project, please consider giving this repo and the docker-hub-repo a Star.

**Comment by moonstarsky37** [comment]
@jammsen Yes, I have seen it. Since I'm not sure where the blocking by the Security app might occur, and I have checked the acf content under the URL you provided, which actually points to an existing location, I think my issue is different. As the discussion and responses here don't seem to resolve my problem, I'll share my own solution. I finally resolved the issue by manually clearing the contents of the steamapps/downloading/* folder and then re-executing the process for it to write successfully.
